src/plugins/ExternalLoaderPlugin.ts

```diff
@@ -0,0 +1,62 @@
+import Compiler = require('webpack/lib/Compiler');
+import Compilation = require('webpack/lib/Compilation');
+import Map from '@dojo/shim/Map';
```

Comment:
Since our support for cli is Node.js 4+ and 4 had `Map` we shouldn't use shim here.  We should instead do like I have done in the [static build PR](https://github.com/dojo/cli-build/pull/172/files#diff-e5e546dd2eb0351f813d63d1b39dbc48) and add the collections lib to `tsconfig.json`